src/adagrams.js

```diff
@@ -1,8 +1,142 @@
+import _, {
+  findLastIndex
+} from "underscore"
+
 const Adagrams = {
   drawLetters() {
     // Implement this method for wave 1
+    const letterPool = [];
+    const addLetters = function (letters, frequency) {
```

Comment:
This is a clever little helper function and a creative way to dynamically fill the `letterPool`.